Header header
uint16 state # State from Tracking.h
uint16 num_closes  # number of loop closure occured

# constants for enum-like access
uint16 UNKNOWN=0
uint16 SYSTEM_NOT_READY=1
uint16 NO_IMAGES_YET=2
uint16 NOT_INITIALIZED=3
uint16 MAPPING=4
uint16 LOCALIZING=5
uint16 LOST=6

